adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -188,6 +189,16 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord])
           conf
         )
     }
+    if (asSingleFile) {
+      log.info("make regular BAM/SAM file (not hadoop)")
```

Comment:
I don't think this is accurate... we can and will want to write single files on Hadoop as well as non-Hadoop, no?

Also `s/regular/single/`.

Maybe:
```
log.info(s"Writing single ${if (asSam) "SAM" else "BAM"} file (not Hadoop-style directory)")
```